packages/reports/tests/integration/components/common-actions/get-api-test.js

```diff
@@ -1,47 +1,73 @@
 import { reject } from 'rsvp';
-import Service from '@ember/service';
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
 import { render, click, findAll } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 import $ from 'jquery';
 
-let Template;
-
-const MockRequest = {
-    serialize: () => 'abc'
-  },
-  MockUrl = 'https://navi.io/api';
+let TemplateElide;
+let TemplateBard;
+let Store;
 
 module('Integration | Component | common actions/get api', function(hooks) {
   setupRenderingTest(hooks);
 
   hooks.beforeEach(function() {
-    this.MockRequest = MockRequest;
+    Store = this.owner.lookup('service:store');
+    this.TestRequestElide = Store.createFragment('bard-request-v2/request', {
+      table: 'tableA',
+      columns: [
+        { field: 'datestamp', parameters: {}, type: 'timeDimension' },
+        { field: 'userCount', parameters: {}, type: 'metric' }
+      ],
+      filters: [],
+      sorts: [],
+      requestVersion: '2.0',
+      dataSource: 'elideOne'
+    });
 
-    Template = hbs`
+    this.TestRequestBard = Store.createFragment('bard-request-v2/request', {
+      table: 'network',
+      filters: [
+        {
+          type: 'timeDimension',
+          field: 'network.dateTime',
+          parameters: { grain: 'day' },
+          operator: 'bet',
+          values: ['current', 'next'],
+          source: 'bardOne'
+        }
+      ],
+      columns: [{ type: 'timeDimension', field: 'network.dateTime', parameters: { grain: 'day' }, source: 'bardOne' }],
+      sorts: [],
+      requestVersion: '2.0',
+      dataSource: 'bardOne'
+    });
+
+    TemplateElide = hbs`
       <CommonActions::GetApi
-        @request={{this.MockRequest}}
+        @request={{this.TestRequestElide}}
+        @buttonClassNames={{this.buttonClassNames}}
+        @beforeAction={{this.beforeAction}}
+      >
+        Get API
+      </CommonActions::GetApi>
+    `;
```

Comment:
We typically try to keep it to one template for test file above the module
```js
const TEMPLATE = hbs`
      <CommonActions::GetApi
        @request={{this.TestRequest}}
        @buttonClassNames={{this.buttonClassNames}}
        @beforeAction={{this.beforeAction}}
      >
        Get API
      </CommonActions::GetApi>
    `;
```

then in the tests do
```js
this.set('TestRequest', this.TestRequestBard);
await render(TEMPLATE);
```